content/en/blog/_posts/2025-05-15-jobs-successpolicy-goes-ga.md

```diff
@@ -1,8 +1,7 @@
 ---
 layout: blog
 title: "Kubernetes 1.33: Job's SuccessPolicy Goes GA"
-date: 2025-04-23
-draft: true
+date: 2025-05-15T10:30:00-08:00
 slug: kubernetes-1-33-jobs-success-policy-goes-ga
 authors: >
```

Comment:
```suggestion
author: >
```

@rytswd Hi, thank you for publishing my blog. It looks like this does not work well.
https://kubernetes.io/blog/2025/05/15/kubernetes-1-33-jobs-success-policy-goes-ga/
<img width="855" alt="Screenshot 2025-05-16 at 9 45 30" src="https://github.com/user-attachments/assets/72399812-9a43-406f-b3d6-a2b46a579046" />